example/testapp/models.py

```diff
@@ -10,9 +10,15 @@ def __unicode__(self):
         return self.plate
 
 
+class m2mprint:
```

Comment:
Get rid of all the changes to this file because they are not needed (you just moved this base model to the test suite).